osbs/cli/main.py

```diff
@@ -352,7 +352,7 @@ def cmd_build(args, osbs):
         required_args = ['platform', 'release']
         unwanted_args = ['platforms']
         create_func = osbs.create_worker_build
-    elif args.orchestrator:
+    elif args.orchestrator and osbs.can_orchestrate():
```

Comment:
Perhaps `get_can_orchestrate` should resolve it? It would override `can_orchestrate` value if `--orchestrate` is set and necessary secrets are specified